test/compatibility/cartridge_upgrade_test.lua

```diff
@@ -7,13 +7,6 @@ local g = t.group()
 
 local helpers = require('test.helper')
 
-local function version(srv)
-    local version = srv:call('require', {'cartridge.VERSION'})
-    local cwd = srv:call('package.loaded.fio.cwd')
-    log.info('Check %s version: %s (from %s)', srv.alias, version, cwd)
-    return version
-end
-
```

Comment:
Please don't remove this code. You can just change `cartridge.VERSION` -> `cartridge.version`.

Reply:
Hmm, as far as I understand, the current check on [scm-1](https://github.com/tarantool/cartridge/actions/runs/4465416213/jobs/7842457371?pr=2056#step:9:24) is no longer [relevant](https://github.com/tarantool/cartridge/actions/runs/4465416213/jobs/7842457371?pr=2056#step:9:24), and the `cartridge_upgrade_test.version` function is also no longer needed

Reply:
> Ok, then remove It and merge PR after the next CI run.

Done, deleted